### PR TITLE
User page and clickable usernames

### PR DESCRIPTION
**Pull request by arrooney**
User is editable from this fragment when you are the owner.

### CODE REVIEW

File: app/src/main/java/com/T05/krowdtrialz/ui/search/SearchActivity.java [hunk 1]
```diff
@@ -92,28 +92,6 @@ public void onSuccess(ArrayList<Experiment> experiments) {
...
-
-                        String experimentID = experiment.getId();
-
-                        i.putExtra("experiment", experimentID);
-                        startActivity(i);
-
-
-
-                    }
-                });
```
Comment (arrooney) [inline app/src/main/java/com/T05/krowdtrialz/ui/search/SearchActivity.java:116]: Note: this was redundant

**Comment by rpshukla** [comment]
Could you explain the purpose of `EditUserDialogFragment` vs `OwnerFragment`?

**Comment by arrooney** [comment]
> 
> 
> Could you explain the purpose of `EditUserDialogFragment` vs `OwnerFragment`?

It's easier to show you. So when I'm at the user screen, or have selected a username from ExperimentDetails I see OwnerFragment:

image


And if I am the user, I can click on the user details and the EditUserDialogFragment pops up:

image

**Comment by rpshukla** [comment]
> > Could you explain the purpose of `EditUserDialogFragment` vs `OwnerFragment`?
> 
> It's easier to show you. So when I'm at the user screen, or have selected a username from ExperimentDetails I see OwnerFragment:
> 
> image
> 
> And if I am the user, I can click on the user details and the EditUserDialogFragment pops up:
> 
> image

Ah that makes sense. Thanks!

**Review by rpshukla — APPROVED** [review]

**Comment by rpshukla** [comment]
Note: we will need to capture the ListenerRegistration returned by `getExperimentsByOwner` when #125 is merged.

**Pull request merged**